components/Voting/sessionPanel.tsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react'
+import { Panel, Icon, Title, Badge, Details, Buttons } from './sessionPanel.styled'
+
+import SessionDetails from '../sessionDetails'
+
+export const SessionPanel: React.FC = ({
```

Comment:
I pulled this branch down to have a look at what was happening - I think one of the reasons is the types are missing here. It'll need an interface and then add that interface to React.FC. `React.FC<INTERFACENAME>`

Reply:
Na, it's because I broke it in the rebase 😬 

But I fixed it now